docs/source/func.api.rst

```diff
@@ -20,3 +20,43 @@ Function Transforms
      jacfwd
      hessian
      functionalize
+
+Utilities for working with torch.nn.Modules
+-------------------------------------------
+
+In general, you can transform over a function that calls a ``torch.nn.Module``.
+For example, the following is an example of computing a jacobian of a function
+that takes three values and returns three values:
+
+.. code-block:: python
+
+    model = torch.nn.Linear(3, 3)
+
+    def f(x):
+        return model(x)
+
+    x = torch.randn(3)
+    jacobian = jacrev(f)(x)
+    assert jacobian.shape == (3, 3)
+
+However, if you want to do something like compute a jacobian over the parameters
+of the model, then there needs to be a way to construct a function where the
+parameters are the inputs to the function.
+That's what :func:`functional_call` is for:
+it accepts an nn.Module, the transformed ``parameters``, and the inputs to the
+Module's forward pass. It returns the value of running the Module's forward pass
+with the replaced parameters.
+
```

Comment:
nit: I know this was how it was like in the original docs, but it would be nice to see the jacrev over parameters example here so that the user doesn't need to click into the functional_call docs.

Reply:
added!